test/test_nestedtensor.py

```diff
@@ -3054,6 +3054,30 @@ def grad_test_func(a, b, c):
 
         gradcheck(grad_test_func, inputs=(a, b, c), check_batched_grad=False)
 
+    def test_unary_pointwise_transposed_inputs(self, device):
```

Comment:
Thanks for the extensive testing! Do you mind also verifying that the contiguity is as expected in the new tests?

Reply:
Added contiguity checks in the tests